internal/frontend/http/ui.go

```diff
@@ -467,7 +467,7 @@ func (f *Frontend) wizardFinal(ctx context.Context, params WizardParams) (string
 			ImageBaseURL: f.options.ExternalURL.JoinPath("image", schematicID, version),
 			PXEBaseURL:   f.options.ExternalPXEURL.JoinPath("pxe", schematicID, version),
 
-			InstallerImage:           fmt.Sprintf("%s/installer/%s:%s", f.options.ExternalURL.Host, schematicID, version),
+			InstallerImage:           fmt.Sprintf("%s/%s-installer/%s:%s", f.options.ExternalURL.Host, params.Platform, schematicID, version),
```

Comment:
1. I think we should only prepend Platform for Talos 1.10+
2. we should update SecureBootInstallImage same way

Reply:
> I think we should only prepend Platform for Talos 1.10+

does it matter, since factory will still handle old `installer` and `installer-secureboot`

Reply:
> does it matter, since factory will still handle old installer and installer-secureboot

I think we shouldn't confuse users with a sudden change which wasn't there